modules/standard/IO.chpl

```diff
@@ -8693,6 +8693,26 @@ proc fileWriter.writeBytes(b: bytes, size = b.size) throws {
   try this.writeBinary(b, size);
 }
 
+private proc sysEndianness() {
```

Comment:
We could even get this to return a `param`, with some effort on the part of the compiler. I would expect that an approach similar to how we compute that C's `int` is `int(32)` would work (basically, generating some Chapel snippet by looking at the C pre-processor output). `runtime/include/qio/bswap.h` has some related code that might be good to draw from.

Reply:
That felt challenging to do in a cross-compilation environment.  At least, more challenging than I wanted to sign up for here given that it's not called in a critical path (so far, at least).